### ISSUE TITLE
TextPicker: onKeyReleased doesn't stop timer on devices where up == menu key

### ISSUE THREAD

**Issue by flocsy** [douglasr/connectiq-samples#4]
Hi

first a bug: on fr965 (and I think on all but at least most 5 key watches) long pressing up == menu.
So in onKeyPressed we get KEY_UP, start the timer, but if we keep it holding long enough then onMenu is called (but of course in onMenu I return longPressTimer != null) and then in onKeyReleased none of the 4 keys are present in the event, but KEY_MENU.

Then a question: what is onKey() for? I tried it with some devices in the simulator but wasn't able to see it ever being called. I'm pretty sure there's a reason why the method is there. Is it needed for some old device?

**Comment by sabeard** [comment]
When I used the text picker in one of my projects, I encountered a similar problem.  On music enabled watches, the long press will eventually launch the music app.  I ended up disabling the long press to scroll through the selection menu.  I eventually added an option to toggle the case and select numbers by adding the selection strings to a 3 position array, then use an index pointer to render the character choices from the array.  One of the menu options allows you to select which index value to use.  With the ability to toggle case and numbers it made the individual button presses needed to scroll through the list not so bad.  So I don't miss the long press to scroll.

There is an open commit request with these modifications to the repository.  It is pull request #2

**Comment by flocsy** [comment]
Yeah, I also have an open PR, but yours is from 4 years ago. I guess it's a waste of time. It's a pity
